FogCreek/Glitch-Community
# Nova chance: Preload optimizely config, track optimizely in react

## Links
https://nova-chance.glitch.me/
https://github.com/FogCreek/Glitch-Community-Work/issues/481

## Changes:
- The server maintains an optimizely instance it uses for server rendering
- The client inits optimizely using the latest data loaded by the server instance
- Renamed state/rollout-toggles to state/optimizely, now exports a useFeatureEnabled hook
- Dried out utils/constants just a little bit

## How To Test:
* Make sure there aren't any errors on team pages, if you're a member or not, and on initial load
